CMakeLists.txt

```diff
@@ -1,277 +1,11 @@
 cmake_minimum_required(VERSION 3.13)
```

Comment:
The root project defines the minimum version and central options (such as the `CXX_STANDARD`)